_data/other.yml

```diff
@@ -194,6 +194,13 @@ websites:
     img: dreamstime.png
     twitter: Dreamstime
 
+  - name: Drift
+    url: https://www.drift.com/
+    img: drift.svg
+    tfa:
+      - totp
+    doc: https://gethelp.drift.com/hc/en-us/articles/360019504214
```

Comment:
```suggestion

```
Probably best to remove this doc link - it doesn't tell a user any useful info about 2FA on their account